### DISCUSSION TITLE
Possible to only mount datasets in certain BEs (e.g. /home)?

### DISCUSSION THREAD

**Discussion by ximGBu4cyQss5P** [zbm-dev/zfsbootmenu#617] [Questions] [answered]
Is it possible to selectively mount datasets depending on what boot environment you are using?  For example, creating a shared /home is easy, and I'd want to do that across a few boot environments with different distros that are all running Xfce.  But I may have other boot environments that are using different major versions of KDE or other DEs, and I might not want that shared /home dataset mounted because of of config differences, wanting different apps to autostart, etc.

**Comment by ahesford** [comment, marked as answer]
Sure, but that's a matter to be configured in the boot environment itself. ZFSBootMenu plays no part in mounting filesystems after it finds and executes your kernel.
